app/services/discovery_report.rb

```diff
@@ -26,12 +25,7 @@ def output_path
 
   # @return [Boolean]
   def using_file_manifest?
-    using_file_manifest && File.exist?(File.join(staging_location, batch.batch_context.file_manifest))
-  end
-
-  # @return [PreAssembly::FileManifest]
-  def file_manifest
-    @file_manifest ||= PreAssembly::FileManifest.new(csv_filename: batch.batch_context.file_manifest, staging_location: staging_location)
+    file_manifest&.exists?
```

Comment:
if we rename the one in the batch class, can we not have this method in other places?

Reply:
I don't know.